server/controllers/projects/getScrums.js

```diff
@@ -0,0 +1,13 @@
+const selectProjectScrum = require('../../database/queries/selectProjectScrums');
+
+exports.getScrums = (req, res, next) => {
+  const { projectId } = req.params;
+  selectProjectScrum(projectId)
+    .then(result => result.rows)
+    .then(result => result.map(ele => ({ id: ele.scrum_id, scrumName: ele.name })))
```

Comment:
@AymanAlqoqa If you want, you could combine these two then-handlers into one:
```
selectProjectScrum(projectId)
    .then(result => result.rows.map(ele => ({ id: ele.scrum_id, scrumName: ele.name })))
```

Reply:
that's right
thanks :+1: